server/index.js

```diff
@@ -1,21 +1,22 @@
-const path = require('path')
```

Comment:
@JoshuaSkootsky it's hard to tell what the actual change is here, since there are lots of linter changes.

Reply:
Can you break this into two commits or two PRS, one that is the linter changes and one that adds helmet?